cas/src/main/java/org/springframework/security/cas/authentication/CasAuthenticationProvider.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004, 2005, 2006 Acegi Technology Pty Limited
+ * Copyright 2004, 2005, 2006, 2024 Acegi Technology Pty Limited
```

Comment:
We don't typically change copyright notices from Acegi, you can leave it as it was.

```suggestion
 * Copyright 2004, 2005, 2006 Acegi Technology Pty Limited
```